packages/ui/src/Layout/AppLayout/index.tsx

```diff
@@ -23,14 +26,13 @@ const AppLayout = () => {
 
   return (
     <div className={styles.viewBox}>
-      <div className={styles.container}>
-        <div className={styles.placeHolder} />
-        <main id="main-form" className={styles.main}>
+      <div className={classNames(styles.container, layoutClassNames.pageContainer)}>
+        {!isMobile && <CustomContent className={layoutClassNames.customContent} />}
```

Comment:
What don't we render the custom content on mobile?

Reply:
Don't have a  design yet, and we are not sure if we gonna support that